tests/performance/local/CMakeLists.txt

```diff
@@ -40,9 +40,9 @@ if(HPX_WITH_EXAMPLES_OPENMP)
      )
 
   set(openmp_homogeneous_timed_task_spawn_FLAGS NOLIBS
-      DEPENDENCIES ${boost_library_dependencies})
-  set(openmp_homogeneous_timed_task_spawn_FLAGS NOLIBS
-      DEPENDENCIES ${boost_library_dependencies})
+      DEPENDENCIES ${boost_library_dependencies} hpx.config hpx.assert)
+  set(openmp_parallel_region_FLAGS NOLIBS
```

Comment:
`hpx.assert` already depends on `hpx.config`. Would it be possible to automatically recursively append dependencies? Otherwise every HPX application will have to list many sub-libraries, which is probably a pain in the end. I think having a top-level dependency `hpx` should be sufficient...

Reply:
I think dependencies should be specified explicitly if they're used directly in the module/application, not transitively (CMake takes care of this case).